.github/workflows/publish-android.yml

```diff
@@ -2,46 +2,84 @@ name: Publish Android Bindings
 on:
   workflow_dispatch:
     inputs:
-      version:
-        description: 'breez-sdk repo release (MAJOR.MINOR.PATCH)'
+      ref:
+        description: 'commit/tag/branch reference'
         required: true
         type: string
+      package-version:
+        description: 'version for the gradle library (MAJOR.MINOR.BUILD)'
+        required: true
+        type: string
+  workflow_call:
+    inputs:
+      ref:
+        description: 'commit/tag/branch reference'
+        required: true
+        type: string
+      package-version:
+        description: 'version for the gradle library (MAJOR.MINOR.BUILD)'
+        required: true
+        type: string
+    secrets:
+      BREEZ_MVN_USERNAME:
+        description: 'username for gradlew publish'
+        required: true
+      BREEZ_MVN_PASSWORD:
+        description: 'password for gradlew publish'
+        required: true
 
 jobs:
-  publish-android:
+  build-package:
     runs-on: ubuntu-latest
-    permissions:
-      contents: read
-      packages: write
```

Comment:
@JssDWt One small thing: Publishing the Android bindings to GitHub packages requires these permissions (at least according to [this guide by GitHub](https://docs.github.com/en/actions/publishing-packages/publishing-java-packages-with-maven#publishing-packages-to-github-packages)). If we see the publishing step to GitHub packages fail, then we'll most likely need to re-add these three lines.

Reply:
Good point. I thought, let me remove some stuff I don't understand the purpose for, and it still worked. So I just left it there. If it breaks, we'll change it.

Reply:
Yep, that's totally valid. Just wanted to give a pointer to make the fix easy in case it does break on release. Great CI improvements overall. :)